src/ODE.jl

```diff
@@ -66,10 +66,10 @@ function hinit(F, x0, t0, tend, p, reltol, abstol)
     # Returns first step, direction of integration and F evaluated at t0
     tdir = sign(tend-t0)
     tdir==0 && error("Zero time span")
-    tau = max(reltol*norm(x0, Inf), abstol)
-    d0 = norm(x0, Inf)/tau
+    tau = max(reltol.*abs(x0), abstol)
```

Comment:
Rather than using abs here, we should allow using a custom norm function. Look at how eg ode23s does this, and pass that norm along to hinit as well.